File: C/Day_7.c
// Get a number from user and subtract 5 to that number and print the result.

#include <stdio.h>

int main(){
    int num, res;
    printf("Enter a number: ");
    scanf("%d", &num);
    res = num - 5;
    printf("Answer: %d", res);
    return 0;
}

File: Java/Day_7.java
// Get a number from user and subtract 5 to that number and print the result.

import java.util.*;
class Main{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int n = scan.nextInt();
        n -= 5;
        System.out.println(n);
    }
}
